src/main/webapp/components/Auth/Auth.js

```diff
@@ -179,9 +179,10 @@ export default function Auth() {
 	    setEmailErrorMsg('Email address has already been used');
 	  }
 	} else {
-	  //dispatch(signIn({ username: username, email: email })
+	  const signInInfo = { username: username, email: email };
+	  console.log(signInInfo);
+	  //dispatch(signIn(signInInfo));
```

Comment:
is the console.log here for debugging? if so, let's remove it
also, is the second line needed?

Reply:
The second line is for later when we implement the react store